modules/tinymce/src/core/main/ts/keyboard/CefNavigation.ts

```diff
@@ -91,8 +92,25 @@ const moveToLineEndPoint = (editor: Editor, forward: boolean): boolean => {
   return NavigationUtils.moveToLineEndPoint(editor, forward, isCefPosition);
 };
 
+const selectToEndPoint = (editor: Editor, forward: boolean): boolean =>
+  getEdgeCefPosition(editor, !forward)
+    .map((pos) => {
+      const rng = pos.toRange();
+      const curRng = editor.selection.getRng();
+      forward ? rng.setStart(curRng.startContainer, curRng.startOffset) : rng.setEnd(curRng.endContainer, curRng.endOffset);
```

Comment:
Ternaries should not be used for setting values. Please convert this to an if/else.
```suggestion
      if (forward) {
        rng.setStart(curRng.startContainer, curRng.startOffset);
      } else {
        rng.setEnd(curRng.endContainer, curRng.endOffset);
      }
```